cachi2/core/requests.py

```diff
@@ -22,24 +22,15 @@
 }
 
 
-def get_requests_session(auth=False, retry_options={}):
+def get_requests_session(retry_options={}):
```

Comment:
Only dropping a note here that CodeQL complained about `import requests` in a  `requests.py` file. I know CodeQL is stupid sometimes, but on the other hand, `requests.py` might also give the wrong first impression that we're talking about a Cachi2 Request.

Definitely not a big deal, but consider if you prefer calling this module `http_client` or `http_requests`.

Reply:
+1, renamed to http_requests